src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int FIGURES_QUANTITY = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+        Figure [] figures = {supplier.getRandomFigure(), supplier.getRandomFigure(),
```

Comment:
create an array of figures with size FIGURES_QUANTITY, by using for loop initialise half of array with random figures and other half with default